frontend/app/src/theme/community/tokyonight.ts

```diff
@@ -0,0 +1,76 @@
+import type { Theme } from "../types";
+
+// Colors
+const white = "#FFFFFF";
+const black = "#000000";
+const raspberry = "#971E57";
+const darkGray = "#696969";
+const silver = "#C0C0C0";
+const veryDarkGray = "#1e2123";
+const lighterDarkGray = "#101010";
+const lightPurple = "#864893";
+const txt = white;
+
+export function getTheme(base: Theme): Theme {
+    // it's ok to mutate the theme passed in because it's a clone
+    base.name = "tokyonight";
+    base.label = "Tokyo Night";
+    base.burst = true;
+    base.bg = "radial-gradient(circle, #69173e 5%, rgb(6 1 22) 100%)";
+    base.bd = darkGray;
+    base.txt = white;
+    base.icon.txt = darkGray;
+    base.icon.selected = raspberry;
+    base.button.bg = raspberry;
+    base.button.hv = raspberry;
+    base.button["hv-txt"] = white;
+    base.button["disabled-txt"] = darkGray;
+    base.button.txt = "rgba(255 255 255 / 80%)";
+    base.button.disabled = silver;
+    base.collapsible.closed.header.txt = txt;
+    base.collapsible.open.header.arrow = white;
+    base.accent = raspberry;
+    base.panel.left.bg = veryDarkGray;
+    base.panel.right.modal = lighterDarkGray;
+    base.modal.bd = darkGray;
+    base.modal.bg = lighterDarkGray;
+    base.chatSummary["bg-selected"] = "rgb(54,59,62 / 50%)";

```

Comment:
this is actually slightly the wrong syntax so this variable is not working. The commas are the problem. Should be: 

```
rgb(54 59 62 / 50%)
```